.github/dependabot.yml

```diff
@@ -8,9 +8,7 @@ updates:
       interval: 'daily'
     labels:
       - "Dependencies"
-      - "Design System Update"
     allow:
       - dependency-name: "@cmsgov/design-system"
       - dependency-name: "@cmsgov/design-system-docs"
       - dependency-name: "@cmsgov/design-system-scripts"
```

Comment:
I'm unsure if we need the set the `allow` for the CMSgov packages

Reply:
Looking at [the docs](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#allow)...

> By default all dependencies that are explicitly defined in a manifest or lock file are kept up to date.

So it doesn't seem like you have to call them out explicitly, but with the current configuration, it also seems like it is catching `@cmsgov` dependencies as well as others

Reply:
I'm not sure if this answers your question, but [it looks like](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#allow) if our `yarn.lock` file defines it, it's kept up to date by default.

> By default all dependencies that are explicitly defined in a manifest or lock file are kept up to date.